strawberryfields/gbs/subgraph.py

```diff
@@ -19,25 +19,43 @@
 
 .. currentmodule:: strawberryfields.gbs.subgraph
 
-This module provides tools for users to identify dense subgraphs.
+This module provides tools for users to search for dense subgraphs of a range of different sizes.
+The search heuristic in this module works by resizing a collection of starting subgraphs and
+keeping track of the densest identified. The starting subgraphs can be selected by sampling from
+GBS, resulting in candidates that are likely to be dense :cite:`arrazola2018using`.
 
-The :func:`search` function provides a heuristic algorithm for finding dense regions and proceeds
-by greedily resizing input subgraphs and keeping track of the densest found.
+An accompanying tutorial can be found :ref:`here <gbs-subgraph-tutorial>`.
+
+Heuristic
+---------
+
+The heuristic algorithm provided proceeds as follows. Each starting subgraph :math:`s` is resized
+to a range of sizes :math:`\{k\}_{k_{\min}}^{k_{\max}}`, resulting in the resized subgraphs
```

Comment:
```suggestion
to a range of sizes :math:`\{k\}_{k_{\min}}^{k_{\max}}`, resulting in the subgraphs
```
Too many "sizes" and "resizes" :)